feedservice/feedservice.go

```diff
@@ -168,11 +168,11 @@ func (f *feed) GetVt(oid string) (models.VT, error) {
 	if timeout != "" {
```

Comment:
Not necessary anymore. Timeout is stored in redis and handled as another script_preferences
https://github.com/greenbone/gvm-libs/pull/578